tests/test_transformers.py

```diff
@@ -10,7 +10,9 @@
                                      Binner,
                                      Renamer,
                                      DateEncoder,
-                                     FreqFeature, DFFeatureUnion)
+                                     FreqFeature,
+                                     DFStandardScaler,
+                                     DFFeatureUnion)
```

Comment:
```suggestion
                                     DFFeatureUnion,
                                     )
```

Samme som tidligere 😄